<launch>
  <node pkg="rover_nav" type="depth_hazard_monitor.py"
        name="depth_hazard_monitor" output="screen">
    <param name="cloud_topic" value="/camera/depth/points"/>
    <param name="base_frame" value="base_link"/>
    <param name="min_x" value="0.3"/>
    <param name="max_x" value="1.0"/>
    <param name="max_y" value="0.5"/>
    <param name="min_z" value="-0.2"/>
    <param name="max_z" value="0.5"/>
    <param name="max_points" value="8000"/>
  </node>
</launch>
